megatron/megatron/arguments.py

```diff
@@ -930,7 +930,8 @@ def _add_training_args(parser):
                        'individual Transformer layers per pipeline stage and do the '
                        'rest without any recomputing at specified granularity'
                        'default) do not apply activations recompute to any layers')
-    group.add_argument('--recompute-num-layers', type=int, default=1,
+    # From dcu
+    group.add_argument('--recompute-num-layers', type=int, default=None,
```

Comment:
I think we don't need to change this default value because this can be set if we want to use it.